web/assets/main.scss

```
@@ -4,3 +4,8 @@ $moj-page-width: 1220px;
 
 @import "node_modules/govuk-frontend/govuk/all";
 @import "node_modules/@ministryofjustice/frontend/moj/all";
+
+.name-grid {
```

Our standard practice is (should be) prefixing with `app-` to distinguish from `govuk-` and `moj-`.
```suggestion
.app-name-grid {
```